server/resources/translations/common.en.json

```diff
@@ -18,6 +18,7 @@
   "sensorDisconnectionInitial": "The Brave Sensor at {{ deviceDisplayName }} ({{ clientDisplayName }}) has disconnected.\nPlease verify that your door sensor is attached and then press the reset button within the circular hole on the ceiling unit using a pin. If you do not receive a reconnection message within the next 15 to 20 minutes after pressing the reset button, contact us by emailing clientsupport@brave.coop.",
   "sensorDisconnectionReminder": "The Brave Sensor at {{ deviceDisplayName }} ({{ clientDisplayName }}) is still disconnected. \nPlease verify that your door sensor is attached and then press the reset button within the circular hole on the ceiling unit using a pin. If you do not receive a reconnection message within the next 15 to 20 minutes after pressing the reset button, contact us by emailing clientsupport@brave.coop.",
   "sensorIsTampered": "The door sensor at {{ deviceDisplayName }} is not fully attached to the door. If you require a replacement door sensor or have any questions about this, contact us by emailing clientsupport@brave.coop.",
+  "sensorFallOff": "The door sensor at {{ deviceDisplayName }} has not given an alert in a long time, please make sure that it is fully attached to the door. If you require a replacement door sensor or have any questions about this, contact us by emailing clientsupport@brave.coop.",
```

Comment:
Please add your translations here for English-FrenchCA translations: https://docs.google.com/spreadsheets/d/1Nna-TF9kwtB3xYhZl2GHrYbExVaYc80P/edit#gid=304727870

And for English-Spanish: https://docs.google.com/spreadsheets/d/1oSWm6Jm0A88X7hrYmH8fjoc-gqpPV9JF/edit#gid=1656917058

Use Google Translate for translating from English, but ping Oona in #sensor-aa-general in Slack and let her know you've added new messages. Also paste your English version there, and other people in the team can give some feedback on the wording

Reply:
added and sent!